oak_restricted_kernel/src/acpi.rs

```diff
@@ -249,6 +249,56 @@ fn name_from_eisa_id(eisa_id: u64) -> Result<String, AmlError> {
 
     Ok(str)
 }
+
+pub struct AcpiDevice {
+    name: AmlName,
+}
+
+impl AcpiDevice {
+    /// Return the value of the `_HID` object, if present.
+    ///
+    /// ACPI specifications require one of `_HID` or `_ADR` to be present.
+    /// See Section 6.1 of the ACPI spec for more details.
+    pub fn hid(&self, ctx: &mut AmlContext) -> Result<Option<String>> {
+        let hid = self.invoke("_HID", ctx);
+
+        if let Ok(hid) = hid {
+            // If the name is an integer, it's a compressed EISA identifier (yes, really)
```

Comment:
super nit:  comments should end with a `.`

Reply:
I'll address that in the next PR as I don't want to spend 40+ minutes waiting for checks :)